chat-client.ts

```diff
@@ -0,0 +1,144 @@
+import memoizeOne from "memoize-one";
+
+import { ClientOptions } from "./types";
+
+const initializeWSClient = memoizeOne(async (projectId: string) => {
+  return new Promise<WebSocket>((resolve, reject) => {
+    const wsClient = new WebSocket("wss://api.commonbase.com/chats");
```

Comment:
If the sessionId is known, it should be added to the WebSocket endpoint. This will be important if a client wants to resume a previous chat session. (we can add this at some point later)